enderio-machines/src/generated/resources/data/enderio_machines/recipes/alloy_smelting/redstone_alloy_ingot.json

```diff
@@ -0,0 +1,16 @@
+{
+  "type": "enderio_machines:alloy_smelting",
+  "ingredients": [
+    {
+      "item": "minecraft:redstone"
+    },
+    {
+      "item": "enderio:silicon"
```

Comment:
use forge/silicon tag as it's used by refined storage (and probably others, also add our silicon to this tag)